DmfSamples/InterfaceSample2/README.md

```diff
@@ -0,0 +1,1344 @@
+InterfaceSample2 Sample KMDF/DMF Function Driver

```

Comment:
KMDF/UMDF

Reply:
no this is correct... KMDF/DMF driver. it is kernel mode driver.